docs/01-overview/main-areas/observability/obsv-04-telemetry-in-kyma.md

```diff
@@ -185,9 +178,9 @@ spec:
 
 ### Step 4: Add authentication details from Secrets
 
-Integrations into external systems usually need authentication details dealing with sensitive data. To handle that data properly in Secrets, the LogPipeline supports the reference of Secrets. The key-value entries of the Secrets can be mapped to environment variables of the Fluent Bit Pods, and with that, are available for usage in [placeholder expressions](https://docs.fluentbit.io/manual/administration/configuring-fluent-bit/classic-mode/variables).
+Integrations into external systems usually need authentication details dealing with sensitive data. To handle that data properly in Secrets, the LogPipeline supports the reference of Secrets.
 
-To leverage data provided by a Kubernetes Secrets in a custom output definition, use placeholder expressions for the data provided by the Secret. Then specify the actual mapping to the keys of the Secret in the `variables` section, like in the following example:
+With the `http` and the `grafana-loki` output definition, the `valueFrom` attribute allows the mapping of Secret keys as visible in the following `http` output example:
```

Comment:
```suggestion
With the **http** and the **grafana-loki** output definition, the **valueFrom* attribute allows for mapping of Secret keys as visible in the following **http** output example:
```
Parameters are formatted as **bold**, their values are formatted as `code font`. 
See the [guidelines](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/03-formatting.md#code-font-and-bold-font) for reference.
Not sure about `grafana-loki`, 'cause I can't see it anywhere in the YAML below.

Reply:
grafana-loki is also a parameter 👍

Reply:
will push it in separate commit since this also applies somewhere else